twitcher/wps_restapi/jobs/jobs.py

```diff
@@ -74,7 +75,9 @@ def check_status(url=None, response=None, sleep_secs=2, verify=False):
         xml = response
     elif url:
         logger.debug('using status_location url ...')
-        xml = requests.get(url, verify=verify).content
+        request_session = requests.Session()
+        request_session.mount('file://', FileAdapter())
```

Comment:
Why are we needing this?

Reply:
When the url was passed, I was getting an error instead of expected behavior.
The request didn't how to handle the file input somehow...